vm/src/stdlib/collections.rs

```diff
@@ -0,0 +1,346 @@
+use crate::function::OptionalArg;
+use crate::obj::{objbool, objsequence, objtype::PyClassRef};
+use crate::pyobject::{IdProtocol, PyClassImpl, PyIterable, PyObjectRef, PyRef, PyResult, PyValue};
+use crate::vm::ReprGuard;
+use crate::VirtualMachine;
+use itertools::Itertools;
+use std::cell::{Cell, RefCell};
+use std::collections::VecDeque;
+
+#[pyclass(name = "deque")]
+#[derive(Debug, Clone)]
+struct PyDeque {
```

Comment:
Are there coming more collection types? If so, it might be handy to create a subfolder with seperate files per type. Not sure if you want to do this at this point.

Reply:
I think that was the only one that needed a native implementation, the rest are implemented by the python collections module. It would be good to eventually optimize and add the native implementations, though.